crawl-ref/source/fight.cc

```diff
@@ -905,6 +905,12 @@ bool bad_attack(const monster *mon, string& adj, string& suffix,
     if (check_landing_only)
         return bad_landing;
 
+    if (you.duration[DUR_LIFESAVING] && mon->holiness() & MH_NATURAL)
```

Comment:
```suggestion
    if (you.duration[DUR_LIFESAVING] && mon->holiness() & (MH_NATURAL | MH_PLANT))
```

Since this is what's actually checked when triggering the conduct.

Reply:
Fair enough...